ProductionModePlugin.js

```diff
@@ -1,5 +1,5 @@
 var CommonJsRequireDependency = require("webpack/lib/dependencies/CommonJsRequireDependency");
-var ConcatSource = require("webpack/lib/ConcatSource");
+var ConcatSource = require("webpack-sources").ConcatSource;
```

Comment:
How does `webpack@1.9.0/lib/ConcatSource` compare to `webpack-sources` please? Is it equivalent or are there any changes? Thanks